configs/pit/pit_xs_ascend.yaml

```diff
@@ -0,0 +1,58 @@
+# system
+mode: 0
+distribute: True
+num_parallel_workers: 8
+val_while_train: True
+
+# dataset
+dataset: 'imagenet'
+data_dir: 'path/to/imagenet'
+shuffle: True
+dataset_download: False
+batch_size: 128
+drop_remainder: True
+
+# augmentation
+image_resize: 224
+scale: [0.08, 1.0]
+ratio: [0.75, 1.333]
+re_value: 'random'
+hflip: 0.5
+interpolation: 'bicubic'
+color_jitter: [0.3, 0.3, 0.3]
+auto_augment: 'randaug-m9-mstd0.5-inc1'
+re_prob: 0.25
+crop_pct: 0.875
+mixup: 0.8
+cutmix: 1.0
+
+# model
+model: 'pit_xs_224'
+num_classes: 1000
+pretrained: False
+ckpt_path: ''
+keep_checkpoint_max: 10
+ckpt_save_dir: './ckpt/'
+epoch_size: 600
+drop_path_rate: 0.1
+dataset_sink_mode: True
+amp_level: 'O2'
+
+# loss
+loss: 'ce'
+label_smoothing: 0.1
+
+# lr scheduler
+scheduler: 'warmup_cosine_decay'
+lr: 0.001
+min_lr: 0.00001
+decay_epochs: 590
+warmup_epochs: 10
+
+# optimizer
+opt: 'adamw'
+filter_bias_and_bn: True
+momentum: 0.9
+weight_decay: 0.05
+dynamic_loss_scale: True
```

Comment:
loss_scale_type: '

Reply:
We use the default "fixed" for loss_scale_type. And add it in yaml for clearness.

Reply:
It has been modified.